assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -188,17 +338,39 @@ export const drawDateRangeControls = function(elem, store, siteno) {
         .attr('ga-event-category', 'TimeSeriesGraph')
         .attr('ga-event-action', d => `changeDateRangeTo${d.period}`)
         .on('change', function() {
+            console.log('ran change')
             const selected = li.select('input:checked');
             const selectedVal = selected.attr('value');
+
+            // Remove any values stored in the form, because they may not match what is shown in the graph until the submit button is pushed
+            store.dispatch(ivTimeSeriesStateActions.setUserInputNumberOfDays(''));
+            store.dispatch(ivTimeSeriesStateActions.setCustomIVTimeRange(null));
+
             if (selectedVal === 'custom') {
-                customDateContainer.attr('hidden', null);
+                // Remove any values stored in the form, because they may not match what is shown in the graph until the submit button is pushed
+                store.dispatch(ivTimeSeriesStateActions.setUserInputNumberOfDays(''));
```

Comment:
This code (that dispatches the actions) can be pulled out of the if clause because it is always executed (it is repeated in the else section)

Reply:
Very true. Fixed